DirectEmployers/MyJobs
# Add loading spinner to ReportRedirect.jsx [PD-3731]

## What's this PR do?
Adds LoadingSpinner to ReportRedirect.jsx
## Screenshots (if appropriate)
As this is across two "pages", there may be a bit of hitching.
![peek 2017-10-11 10-34](https://user-images.githubusercontent.com/3379265/31497180-37575958-af2c-11e7-93ae-ffb640af28e4.gif)
## Where should the reviewer start?
ReportRedirect.jsx
## How should this be manually tested?
Open up reporting. Navigate around. In PRM reporting, change your report category. Note that, in all cases that ReportRedirect is used, we display a loading spinner.
## Any background context you want to provide?
-
## PR Checklist

### Front-End
